pkgs/applications/networking/sniffers/wireshark/default.nix

```diff
@@ -157,6 +157,10 @@ stdenv.mkDerivation rec {
 
   cmakeFlags =
     [
+      # the cmake package does not handle absolute CMAKE_INSTALL_INCLUDEDIR correctly
+      # (setting it to an absolute path causes include files to go to $out/$out/include,
+      #  because the absolute path is interpreted with root at $out).
```

Comment:
The comment is somewhat misleading, implying that the current `dev` output has headers at the wrong location, which isn't true.

Change to something like "look for the headers in $out/$out/include" instead if that's what you mean.